api/src/main/java/module-info.java

```diff
@@ -172,13 +177,14 @@
  *
  * <h2>Methods with Entity Parameters</h2>
  *
- * <p>You can annotate a method with {@link Insert}, {@link Update}, {@link Save},
- * or {@link Delete} if the method accepts a single parameter, which must be one of:</p>
+ * <p>A method may be annotated with a lifecycle annotation such as {@link Insert},
+ * {@link Update}, {@link Save}, or {@link Delete}. The method must accept a single
+ * parameter, whose type is either:</p>
```

Comment:
Updating to make it clear that the Life cycle annotation is required and not optional on any methods that have an entity parameter.
```suggestion
 * <p>A method must be annotated with a lifecycle annotation such as {@link Insert},
 * {@link Update}, {@link Save}, or {@link Delete} if the method accepts a single
 * parameter, whose type is either:</p>
```

Reply:
Please see c037289994158fbb13dcc8cf0e9140b1710489e0.